### PR TITLE
Thrive chunk improvements

### PR DESCRIPTION
**Pull request by Untrustedlife**
fixes https://github.com/Revolutionary-Games/Thrive/issues/778

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
test this please

**Comment by hhyyrylainen** [comment]
test this please (trying again, Jenkins is being really silly with logs again).

File: scripts/microbe_stage/biome.as [hunk 1]
```diff
@@ -126,12 +124,25 @@ ObjectID createChunk(CellStageWorld@ world, uint chunkId,  Float3 pos)
...
+
+    //chunk properties
+    if (chunk.damages > 0.0f || chunk.deleteOnTouch){
+        auto damager = world.Create_DamageOnTouchComponent(chunkEntity);
+        damager.setDamage(chunk.damages);
+        damager.setDeletes(chunk.deleteOnTouch);
+        auto body = rigidBody.CreatePhysicsBody(world.GetPhysicalWorld(),
+            world.GetPhysicalWorld().CreateSphere(radius),mass,
+        //damage
+        world.GetPhysicalMaterial("chunkDamageMaterial"));
```
Comment (hhyyrylainen) [inline scripts/microbe_stage/biome.as]: What does this do? Shouldn't the return value of this function be used somewhere???

Comment (Untrustedlife) [reply scripts/microbe_stage/biome.as]: You must be tired,  that’s part of the create physics method. See coma on line before comment

Comment (hhyyrylainen) [reply scripts/microbe_stage/biome.as]: I'm not going to accept that I'm blind. That line should obviously be indented as it's part of expression on the previous line.

Comment (Untrustedlife) [reply scripts/microbe_stage/biome.as]: I agree about that

Comment (Untrustedlife) [reply scripts/microbe_stage/biome.as]: Okay i have fixed this :)

**Comment by Untrustedlife** [comment]
i fxed the issue you had with it, im not sure why clang is failing, i ran the formatting script. And thrivetest is passing all tests.

**Comment by hhyyrylainen** [comment]
The formatting is failing because of the CSS file:
```
scripts/gui/thrive_style.css
 1213:3  ✖  Unexpected duplicate "font-size"               declaration-block-no-duplicate-properties
 1235:3  ✖  Unexpected duplicate "color"                   declaration-block-no-duplicate-properties
 1237:3  ✖  Unexpected duplicate "font-size"               declaration-block-no-duplicate-properties
 1424:6  ✖  Unexpected unknown type selector "toolhead"    selector-type-no-unknown                 
 1433:6  ✖  Unexpected unknown type selector "tkey"        selector-type-no-unknown                 
 1441:6  ✖  Unexpected unknown type selector "toolgold"    selector-type-no-unknown                 
 1449:6  ✖  Unexpected unknown type selector "toolbonus"   selector-type-no-unknown                 
 1457:6  ✖  Unexpected unknown type selector "toolmalus"   selector-type-no-unknown                 
 1465:6  ✖  Unexpected unknown type selector "toolbody"    selector-type-no-unknown                 
 1472:6  ✖  Unexpected unknown type selector "tooldesc"    selector-type-no-unknown
```

**Comment by Untrustedlife** [comment]
Our custom spans are perfectly valid CSS it’s just not used as often . So that’s an issue with clang

**Comment by hhyyrylainen** [comment]
Making "custom" HTML tags is not something that I've ever seen before. I don't think it's good style. All of the custom types should be converted to <span class="..."> and the CSS rules prefixed with `.` to fix them.

**Comment by Untrustedlife** [comment]
According to stack overflow it sconsidered best practice nowadays:
https://stackoverflow.com/questions/37622348/is-it-bad-practice-to-define-a-custom-html-tag

But if i have to do it i will lol.

**Comment by Untrustedlife** [comment]
Okay pushed the fix

**Pull request merged**